<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossplotVolumesInputDialog</class>
 <widget class="QDialog" name="CrossplotVolumesInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X-Axis:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbVolumeX"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y-Axis:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbVolumeY"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="cbAttribute">
       <property name="text">
        <string>Display Attribute</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAttribute">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Attribute:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbVolumeAttribute">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CrossplotVolumesInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CrossplotVolumesInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAttribute</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbVolumeAttribute</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAttribute</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelAttribute</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
